CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.7.0
+
+* Feature - Allow callers to set log output. ([#309](https://github.com/awslabs/amazon-ecr-credential-helper/pull/309) and [#312](https://github.com/awslabs/amazon-ecr-credential-helper/pull/312))
+* Upgrade dependencies for bug fixes and security patches. 
+* Minimum supported go version changed from 1.13 to 1.15. ([#336](https://github.com/awslabs/amazon-ecr-credential-helper/pull/336))
```

Comment:
End user probably does not care the min Go version since we are distributing the binary.

Reply:
Good call out.  I didn't think much that the changelog will be used as part of a package distribution. 
I will remove this part.